### PR TITLE
Rate limit UI updates to avoid browser death

### PR DESCRIPTION
**Pull request by dbkr**
Requires https://github.com/matrix-org/matrix-react-sdk/pull/139

Fixes https://github.com/vector-im/vector-web/issues/779

### CODE REVIEW

File: src/components/structures/RightPanel.js [hunk 1]
```diff
@@ -66,15 +67,19 @@ module.exports = React.createClass({
...
         else if (this.state.phase === this.Phase.MemberInfo && member.roomId === this.props.roomId &&
                 member.userId === this.state.member.userId) {
             // refresh the member info (e.g. new power level)
-            this.forceUpdate();
+            this._delayedUpdate();
         }
     },
 
+    _delayedUpdate: new rate_limited_func(function() {
+        this.forceUpdate()
```
Comment (Kegsay) [inline src/components/structures/RightPanel.js]: Is this the `this` you're loooking for? Also, missing `;`

**Comment by Kegsay** [comment]
LGTM aside from comment.

File: src/components/structures/RightPanel.js (hunk 1 above)
Comment (dbkr) [inline src/components/structures/RightPanel.js]: Yeah, I made the rate_limited_func function so you could still use this as you expect, so it works out.

**Comment by Kegsay** [comment]
LGTM

**Pull request merged**